sources/packages/web/src/components/common/LocationSummary.vue

```diff
@@ -129,10 +129,13 @@ export default {
     };
 
     const addressList1 = (item: InstitutionLocationsDetails) => {
+      let addressString = item.data.address.provinceState
```

Comment:
Please rename the addressString variable to stateCityPostalCodeString or something. 😄  I know its tough to name these. Just a suggestion.